src/features/ValueEditor/ValueEditor.tsx

```diff
@@ -141,13 +144,17 @@ export const ValueEditor = ({
           )}
         </Flex>
         <Flex gap="sm">
-          <Tooltip label="Add value">
+          <Tooltip label="Clear value">
             <ActionIcon
-              disabled={onAdd == undefined}
               variant="light"
-              aria-label="Add value"
-              onClick={() => onAdd?.()}
+              aria-label="Clear value"
+              onClick={() => onClear(index)}
             >
+              <FiTrash />
+            </ActionIcon>
+          </Tooltip>
+          <Tooltip label="Add value">
+            <ActionIcon variant="light" aria-label="Add value" onClick={onAdd}>
```

Comment:
I've removed the possibility for the `add` button to be disabled for the reasons explained above: https://github.com/marilari88/zod-playground/pull/24/files#r1594704800